src/BladeSDK.ts

```diff
@@ -2233,6 +2232,80 @@ export class BladeSDK {
         }
     }
 
+    /**
+     * Emergency balance transfer from broken mnemonic account to new account
+     * Accounts with broken mnemonic sometimes were created because of hedera-sdk issue
+     * To transfer funds from broken mnemonic account to new account a couple of steps required:
+     * 1. Create new account
+     * 2. Associate all tokens with new account that you want to transfer
+     * 3. Call this method to transfer funds to new account
+     * 4. Send some HBAR to broken mnemonic account to cover fees if needed
+     *
+     * @param seedPhrase mnemonic from account
+     * @param accountId account id (broken)
+     * @param receiverId new account id
+     * @param hbarAmount amount of HBAR to resque. Can be 0
+     * @param tokenList list of token ids to transfer all tokens. Can be empty
+     * @param checkOnly if true, will only check if mnemonic is broken. No transfer will be made
+     * @param completionKey optional field bridge between mobile webViews and native apps
+     * @returns {EmergencyTransferData}
+     * @example
+     * const receipt = await bladeSdk.brokenMnemonicEmergencyTransfer(brokenSeed, resqueAccountId, newAccountId, "0.5", ["0.0.1337"], false);
+     */
+    async brokenMnemonicEmergencyTransfer(seedPhrase: string, accountId: string, receiverId: string, hbarAmount: string, tokenList: string[], checkOnly: boolean, completionKey?: string): Promise<EmergencyTransferData> {
```

Comment:
Are we sure that we want to combine check and fix methods into one?

Reply:
yes, KC firstly going to collect some statistics, and after that use it (or not).
there is no need to create separate methods for that temporary feature